snapcraft/projects.py

```diff
@@ -270,7 +269,7 @@ class Project(ProjectModel):
     epoch: Optional[str]
     adopt_info: Optional[str]
     system_usernames: Optional[Dict[str, Any]]
-    environment: Optional[Dict[str, str]]
+    environment: Optional[Dict[str, Any]]
```

Comment:
this might be better kept as a str as environment variable values are just that, strings

Reply:
Good catch.